### PR TITLE
Implement component CartSummaryInfo

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
Implement component CartSummaryInfo

image

storybook: https://6684faab2d0c99a3e93bf145-bjzygtxjyd.chromatic.com/

### CODE REVIEW

File: src/components/Checkout/Checkout.tsx [hunk 1]
```diff
@@ -1,9 +1,28 @@
...
 interface CheckoutProps {
   onClick: () => void;
+  numberProduct: number;
+  totalPrice: number;
 }
 
-const Checkout = (_: CheckoutProps) => {
-  return <></>;
-};
+const Checkout = ({ numberProduct, totalPrice, onClick }: CheckoutProps) => (
```
Comment (thanhnguyen-agilityio) [inline src/components/Checkout/Checkout.tsx]: component `CartSummaryInfo`

Comment (thanhnguyen-agilityio) [inline src/components/Checkout/Checkout.tsx]: `onCheckout` or `onClickCheckoutButton`

File: src/components/Checkout/Checkout.tsx [hunk 2]
```diff
@@ -1,9 +1,28 @@
...
+  <div className={styles.container}>
+    <p className={styles.product}>{numberProduct} Product</p>
+    <div className={`${utils.flexCenter} ${styles.total}`}>
+      <p className={styles.desc}>Total</p>
```
Comment (thanhnguyen-agilityio) [inline src/components/Checkout/Checkout.tsx]: Use `label`

File: src/components/Checkout/Checkout.tsx (hunk 1 above)
Comment (thanhtrantruong47) [reply src/components/Checkout/Checkout.tsx]: Fixed

File: src/components/Checkout/Checkout.tsx (hunk 2 above)
Comment (thanhtrantruong47) [reply src/components/Checkout/Checkout.tsx]: Fixed

File: src/components/Checkout/Checkout.tsx (hunk 1 above)
Comment (thanhtrantruong47) [reply src/components/Checkout/Checkout.tsx]: Fixed

**Review by thanhnguyen-agilityio — APPROVED** [review]

**Pull request merged**